base/update-lthooks-tests.sh

```diff
@@ -34,7 +34,7 @@ l3build save -cconfig-lthooks2 \
 
 
 
-l3build save -cconfig-lthooks \
+L3build save -cconfig-lthooks \
```

Comment:
```suggestion
l3build save -cconfig-lthooks \
```

Reply:
ouch :-) comes from using ^C in emacs instead of the ^W